updater/lib/dependabot/dependency_snapshot.rb

```diff
@@ -132,7 +121,7 @@ def handled_dependencies
 
     # rubocop:disable Performance/Sum
     sig { returns(T::Set[String]) }
-    def handled_group_dependencies
+    def handled_dependencies_all_directories
```

Comment:
🤔 `_all_directories` reads as being very open to interpretation. I could see this signaling:

* That the dependencies are mapped to all directories (a product-like result)
* That the dependencies are returned agnostic of their directory 👈
* That the dependencies are combined with their directories (and/or all directories)

From context, I suspect it's the one with 👈 next to it.

Are `directories` important here? If they are, I wonder if there's another signal we could use to make it unambiguous.

Reply:
Naming is hard! 

In #9610 what I was doing was rename all the directory-contextually-aware code to have a `_current_directory` suffix, would that be clearer? Then we can leave this as `handled_dependencies` or even `all_handled_dependencies`?